### ISSUE TITLE
5.8.8 failed to install: no amd64 directory

### ISSUE THREAD

**Issue by anibyl** [bkw777/mainline#60]
```
Preparing to install '5.8.8'
/ust/bin/env/: cannot change directory to'/home/anibyl/.cache/mainline/5.8.8/amd64': No such file or directory
E: Installation completed with errors
```

![image](https://user-images.githubusercontent.com/790717/92949888-aa2dbe00-f410-11ea-89b6-d17877d2bb5e.png)

Mainline 1.0.12, Ubuntu 20.04, Linux 5.8.7

5.8.8 amd64 build is available in Ubuntu kernels: https://kernel.ubuntu.com/~kernel-ppa/mainline/v5.8.8/

**Comment by Aisyk** [comment]
Same thing here
![Capture d’écran du 2020-09-11 22-41-26](https://user-images.githubusercontent.com/5584650/92971011-f82dda80-f47f-11ea-80e5-621e129999a8.png)
Mainline 1.0.12, PopOS  20.04, Linux 5.8.3

**Comment by Aisyk** [comment]
When i create the directory... it seems to have a problem with dpkg...

![Capture d’écran du 2020-09-12 11-40-45](https://user-images.githubusercontent.com/5584650/92992664-d585db00-f4ec-11ea-9f8f-e411c109fc69.png)

**Comment by Aisyk** [comment]
Now, when Mainline wants to download the last release (5.8.9)
![Capture d’écran du 2020-09-12 20-33-57](https://user-images.githubusercontent.com/5584650/93002440-684b6780-f537-11ea-9135-8c7a3ba3122c.png)

**Comment by anibyl** [comment]
5.8.9 installation works for me.

**Comment by Aisyk** [comment]
Ok, i re-try now, installation works with no problem.

**Comment by miguelzabala** [comment]
I've the same problem.

Linux Mint 20. 

![image](https://user-images.githubusercontent.com/994576/94156215-3949c580-fe56-11ea-8f67-9893cda46824.png)

**Comment by miguelzabala** [comment]
Removing index.html from the folder solved it for me. Removed the file and refreshed then I was able to download it and install it.

**Comment by skyfaller** [comment]
@miguelzabala Removing index.html fixed it for me too! Thanks for finding a workaround.

**Comment by bkw777** [comment]
Curious. Well thanks for finding a work-around at least until the real bug is fixed.

**Comment by user85475485** [comment]
![image](https://user-images.githubusercontent.com/64977518/94527668-9d460280-0237-11eb-9522-8dcc5270ebeb.png)
Same thing here on Linux Mint 20

**Comment by zlamalp** [comment]
This can happen randomly for any kernel, since it offers kernels based on the index file - eg. before they are actually build in the ppa for your architecture. Then it stays in this inconsistent state, unless you remove ```~/.cache/mainline/<version>/```

**Comment by user85475485** [comment]
Hey,
Thx for the answer i fixed it already by doing sudo mainline --clean-cache


Am Sa., 3. Okt. 2020 um 11:00 Uhr schrieb Pavel Zlámal <
notifications@github.com>:

**Comment by bkw777** [comment]
> This can happen randomly for any kernel, since it offers kernels based on the index file - eg. before they are actually build in the ppa for your architecture. Then it stays in this inconsistent state, unless you remove `~/.cache/mainline/<version>/`

Ahh thank you for figuring that out.

**Comment by mmiat** [comment]
same with 5.9.3 on Mint 20

**Comment by skyfaller** [comment]
I am now unable to make any of the workarounds work for 5.9.5.

**Comment by zlamalp** [comment]
> I am now unable to make any of the workarounds work for 5.9.5 or 5.9.6.

If you look into the PPA itself, you can see, that build of kernel 5.9.5 for amd64 failed, hence you can't install it at all.
For 5.9.6 it was build like seconds ago, so it should work now.

See: https://kernel.ubuntu.com/~kernel-ppa/mainline/v5.9.6/

**Comment by skyfaller** [comment]
aha, 5.9.6 does work for me now :) I will have to look at the ppa itself next time, thank you.

**Comment by anibyl** [comment]
5.9.9 failed to install. Package is not listed in `dpkg --list` neither before nor after the reboot.
![image](https://user-images.githubusercontent.com/790717/99634140-d5c2bb00-29f4-11eb-9235-27bd7a85e134.png)

**Comment by bkw777** [comment]
> 5.9.9 failed to install. Package is not listed in `dpkg --list` neither before nor after the reboot.
> ![image](https://user-images.githubusercontent.com/790717/99634140-d5c2bb00-29f4-11eb-9235-27bd7a85e134.png)

There is no 5.9.9. It failed to build.

**Comment by eliotb** [comment]
As noted in the previous comment, sometimes a kernel fails to build.  In this case, is there a way to filter it out of the list of offered kernels?  Consider this a feature request ;)

**Comment by AMV007** [comment]
mainline 1.0.14
Distribution: Ubuntu 20.04.2 LTS
Architecture: amd64
Running kernel: 5.8.0-59-generic
E: Internet connection is not active
Fetching index from kernel.ubuntu.com...
OK
----------------------------------------------------------------------
Found installed: 5.4.0-77.86
Found installed: 5.8.0-43.49~20.04.1
Found installed: 5.4.0.77.80
Found installed: 5.8.0-59.66~20.04.1
Found installed: 5.8.0.59.66~20.04.42
----------------------------------------------------------------------
----------------------------------------------------------------------
Preparing to install '5.11.16'
/usr/bin/env: cannot change directory to '/root/.cache/mainline/5.11.16/amd64': No such file or directory
E: Installation completed with errors


but I installed this version one month before at another PC ?

**Comment by bkw777** [comment]
fixed wih 06081aaafd025e36b886ea04a788d28a317fc81a
